app/partials/data-export-button.cjsx

```diff
@@ -0,0 +1,53 @@
+React = require 'react'
+PromiseRenderer = require '../components/promise-renderer'
+apiClient = require '../api/client'
+moment = require 'moment'
+Translate = require 'react-translate-component'
+
+module.exports = React.createClass
+  displayName: 'DataExportButton'
+
+  getInitialState: ->
+    exportRequested: false
+    exportError: null
+
+  exportGet: ->
+    @props.project.get @props.exportType
+      .catch ->
+        []
+
+  requestDataExport: ->
+    @setState exportError: null
+    apiClient.post @props.project._getURL(@props.exportType), media: content_type: 'text/csv'
+      .then =>
+        @classificationsExportGet = null
```

Comment:
I think this was left over from the old page, and it looks like the request is no longer actually being cached, so a project without these resources makes the request every time the component renders which is every time the project resource changes.